backend/modules/email/subModules/emailMeta/services/email.meta.service.ts

```diff
@@ -0,0 +1,16 @@
+import EmailMetaDao from "../daos/email.meta.dao";
+import { GetEmailByMetaId } from "../dto/get.email.by.meta.dto";
+import { UpdateEmailMeta } from "../dto/update.email.meta.dto";
+
+class EmailMetaService {
+  
+  async updateEmailMeta(resource: UpdateEmailMeta) {
```

Comment:
get variable name of serviice functions consistent - resourcce/data/payload. think of a better name